src/main/kotlin/app/revanced/utils/adb/Enums.kt

```diff
@@ -0,0 +1,7 @@
+package app.revanced.utils.adb
+
+enum class ROOT(val value: String) {

```

Comment:
```suggestion
enum class ROOT(val name: String) {
```

Reply:
This wouldn't work as name is already a member of the Enum type.

Reply:
Correct, I forgot about that. You could keep it like this then, it's fine.